mithril-test-lab/mithril-devnet/README.md

```diff
@@ -169,7 +169,7 @@ artifacts
 ├── cardano-node
 ├── docker-compose.yaml
 ├── log.sh
-├── node-bft1
+├── node-pool1
```

Comment:
There should be a `node-full1` folder also?

Reply:
With the default configuration, there are `node-pool1`, `node-pool2` and `node-pool3` directories.
Maybe there's something wrong in the script. I'm gonna investigate 👀